src/flags_menu.cpp

```diff
@@ -4,39 +4,41 @@
 #include "utils.h"
 #include "libtp_c/include/flag.h"
 
-#define LINES 10
+#define LINES 11
 
 static Cursor cursor = {0, 0};
 bool init_once = false;
 bool flags_menu_visible;
 
 bool boss_flag;
+bool rupee_flag;
 bool midna_charge;
 bool transform_warp;
 bool midna_on_z;
 bool epona_stolen;
 bool epona_tamed;
-bool map_warping; 
+bool map_warping;
 bool midna_healthy;
 bool midna_on_back;
 bool wolf_sense;
 
 Line lines[LINES] = {
-    {"boss flag", BOSS_FLAG_INDEX, "Set the boss flag value. Press A to lock the value", true, &boss_flag},
-    {"epona stolen", EPONA_STOLEN_INDEX, "Toggle flag for Epona being stolen", true, &epona_stolen},
-    {"epona tamed", EPONA_TAMED_INDEX, "Toggle flag for Epona being tamed", true, &epona_tamed},
-    {"map warping", MAP_WARPING_INDEX, "Toggle flag for having map warping", true, &map_warping},
-    {"midna charge", MIDNA_CHARGE_INDEX, "Toggle flag for Midna charge", true, &midna_charge},
-    {"midna healthy", MIDNA_HEALTHY, "Toggle flag for Midna being healthy/sick", true, &midna_healthy},
-    {"midna on back", MIDNA_ON_BACK, "Toggle flag for Midna appearing on Wolf Link's back", true, &midna_on_back},
-    {"midna on z", MIDNA_Z_INDEX, "Toggle flag for being able to use Midna", true, &midna_on_z},
-    {"transform/warp", TRANSFORM_WARP_INDEX, "Toggle flag for transforming/warping", true, &transform_warp},
-    {"wolf sense", WOLF_SENSE_INDEX, "Toggle flag for having wolf sense", true, &wolf_sense}};
+    {"boss flag", BOSS_FLAG_INDEX, "set the boss flag value. press a to lock the value", true, &boss_flag},
+    {"rupee cutscenes", RUPEE_CS_FLAG_INDEX, "toggle flag for rupee cutscenes being enabled", true, &rupee_flag},
```

Comment:
All of the existing lines should not be changed. The new line should start with a capital letter to match the updated style: "Toggle flag for rupee...."